### PR TITLE
Data Dictionanry and Folder based migration

### PR DESCRIPTION
**Pull request by yingliao422**
- Update admin_console_lambda.template
- update for data dictionary
- update admin console repo with data dictionary support
- update admin console repo with data dictionary support
- update admin console repo with data dictionary support
- update admin console repo with data dictionary support
- fix lambda schedule and handler
- Update README.md
- Update README.md
- Sample csv files
- QuickSight Refresh Optimizer PR
- updated .gitignore
- Update README.md
- cleanup & minor updates
- Delete Dataset_Refresh_Optimizer directory
- Adding QS dataset refresh optimizer delivery kit
- data dict and folder based migration

### CODE REVIEW

File: Migration-scripts/Jupyter Notebook/Functions.ipynb [hunk 1]
```diff
@@ -966,22 +983,48 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "def create_group_membership(session, folderid, objectid, objecttype):\n",
```
Comment (ywang103) [inline Migration-scripts/Jupyter Notebook/Functions.ipynb:996]: you might want to rename it as create_folder_membership.

File: Migration-scripts/Jupyter Notebook/Functions.ipynb [hunk 2]
```diff
@@ -1221,7 +1455,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.4"
+   "version": "3.6.13"
```
Comment (ywang103) [inline Migration-scripts/Jupyter Notebook/Functions.ipynb:1458]: please keep it locally.

File: Migration-scripts/Jupyter Notebook/Functions.ipynb [hunk 3]
```diff
@@ -1207,9 +1441,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "conda_python3",
    "language": "python",
-   "name": "python3"
+   "name": "conda_python3"
```
Comment (ywang103) [inline Migration-scripts/Jupyter Notebook/Functions.ipynb:1446]: please keep it locally.

**Review by ywang103 — CHANGES_REQUESTED** [review]
Please make the changes by the comments. Thanks

**Comment by yingliao422** [comment]
Added a new commit to my fork to address the change requests. Please review again.

**Review by ywang103 — APPROVED** [review]

**Pull request merged**